plugin/src/main/java/io/snyk/languageserver/protocolextension/messageObjects/scanResults/Issue.java

```diff
@@ -4,20 +4,23 @@
 
 public record Issue(String id, String title, String severity, String filePath, Range range, boolean isIgnored,
 		boolean isNew, String filterableIssueType, IgnoreDetails ignoreDetails, AdditionalData additionalData) {
+
 	public String getDisplayTitle() {
 		if (title == null || title.isEmpty()) {
 			return additionalData != null ? additionalData.message() : null;
 		}
-		String displayTitle = title;
+
+		StringBuilder displayTitle = new StringBuilder(title);
+
```

Comment:
This is completely unnecessary - the Java compiler takes care of this. It was different in early JDKs, but nowadays there shouldn't be a difference. I'd vote for readability and deactivate the linting rule.